### PR TITLE
Add iskra documentation

### PR DESCRIPTION
**Pull request by iskrakranj**
## Proposed change

Adds documentation for iskra integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121488
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5620

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Added comprehensive guide for integrating Iskra energy meters and power quality analyzers with Home Assistant.
  - Included configuration options for Modbus TCP and Iskra's Smart Gateway with REST API.
  - Detailed monitored data including power, current, voltage for each phase, and energy counters.
  - Provided specific notes on device configuration and data behavior based on user settings through MiQen software.

### CODE REVIEW

File: source/_integrations/iskra.markdown [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- **Home Assistant**: Add iskra integration, enter Smart Gateway's **IP address** and select **RestAPI** If authentication is required Home Assistant will prompt you to enter Smart Gateway's **credentials**. All devices configured on Smart Gateway will be automatically added to your Home Assistant.
+
+
+### Modbus TCP Connection
+
+If your device supports a direct internet connection, such as PQ meters (iMC/MC series/ iMT/MT series usually), you can use Modbus TCP:
+
+- **Device**: Find your device using [MiQen](https://www.iskra.si/sl/Programska-oprema/MiQen/) software and configure it so it uses static IP.
+- **Home Assistant**: Add iskra integration, enter Devices's **IP address** and select **Modbus TCP**. Home assistant will prompt you to enter Modbus TCP port and Modbus address of your device.
+
```
[the remark this answers is not in the document]
Comment (iskrakranj) [reply source/_integrations/iskra.markdown]: This is selected in Home Assistant integration as which type of connection you want to use to integrate devices not  configuration of Smart Gateway

[the remark this answers is not in the document]
Comment (iskrakranj) [reply source/_integrations/iskra.markdown]: yes, please provide revised version

File: source/_integrations/iskra.markdown [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+## Configuration Options
+
+There are two ways to configure your devices with Home Assistant:
+
+### Smart Gateway with REST API
+
+If your device supports Modbus RTU over RS485/IR, you can use Iskra's Smart Gateway to connect them via REST API:
+
+- **Smart Gateway**: Connect your devices to the Smart Gateway and add your devices to Smart Gateway's configuration. It's also recommended to set static IP on your smart gateway.
+- **Home Assistant**: Add iskra integration, enter Smart Gateway's **IP address** and select **RestAPI** If authentication is required Home Assistant will prompt you to enter Smart Gateway's **credentials**. All devices configured on Smart Gateway will be automatically added to your Home Assistant.
```
Comment (iskrakranj) [inline source/_integrations/iskra.markdown]: User must select RestAPI connection type in Home assistant integration not on Smart Gateway as RestAPI is provided by Smart Gateway device

File: source/_integrations/iskra.markdown [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ha_domain: iskra
+ha_iot_class: local_polling
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+# Iskra Integration
+
+## Description
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
```

File: source/_integrations/iskra.markdown [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- WM series ([WM3XX / WM1XX](https://www.iskra.eu/en/Iskra-Energy-meters/))
+
+### Power quality analyzers
+
+These devices typically support Ethernet connections and use Modbus TCP for data polling:
+
+- iMT/MT series ([MTXXX / iMTXXX](https://www.iskra.eu/en/NEW_SERIES_Universal_measuring_devices_/))
+- iMC/MC series ([MCXXX / iMCXXX](https://www.iskra.eu/en/NEW_SERIES_Universal_measuring_devices_/))
+
+## Configuration Options
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
## Configuration options
```

File: source/_integrations/iskra.markdown [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+### Smart Gateway with REST API
+
+If your device supports Modbus RTU over RS485/IR, you can use Iskra's Smart Gateway to connect them via the REST API:
+
+- **Smart Gateway**: Connect your devices to the Smart Gateway and add your devices to the Smart Gateway's configuration. It's also recommended to set static IP on your smart gateway.
+- **Home Assistant**: Add the iskra integration, enter Smart Gateway's **IP address** and select **RestAPI** as the connection type within the Home Assistant integration. If authentication is required Home Assistant will prompt you to enter Smart Gateway's **credentials**. All devices configured on Smart Gateway will be automatically added to your Home Assistant.
+
+
+### Modbus TCP Connection
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
### Modbus TCP connection
```

File: source/_integrations/iskra.markdown [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+If your device supports a direct internet connection, such as PQ meters (iMC/MC series/ iMT/MT series usually), you can use Modbus TCP:
+
+- **Device**: Find your device using the [MiQen](https://www.iskra.si/sl/Programska-oprema/MiQen/) software and configure it to use static IP.
+- **Home Assistant**: Add the iskra integration, enter the devices's **IP address** and select **Modbus TCP** as the connection type within the Home Assistant integration. Home Assistant will prompt you to enter Modbus TCP port and Modbus address of your device.
+
+{% include integrations/config_flow.md %}
+
+## Monitored Data
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
## Monitored data
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization) also in titles

File: source/_integrations/iskra.markdown [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+## Configuration Options
+
+There are two ways to configure your devices with Home Assistant:
+
+### Smart Gateway with REST API
+
+If your device supports Modbus RTU over RS485/IR, you can use Iskra's Smart Gateway to connect them via the REST API:
+
+- **Smart Gateway**: Connect your devices to the Smart Gateway and add your devices to the Smart Gateway's configuration. It's also recommended to set static IP on your smart gateway.
+- **Home Assistant**: Add the iskra integration, enter Smart Gateway's **IP address** and select **RestAPI** as the connection type within the Home Assistant integration. If authentication is required Home Assistant will prompt you to enter Smart Gateway's **credentials**. All devices configured on Smart Gateway will be automatically added to your Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
- **Home Assistant**: Add the Iskra integration, enter Smart Gateway's **IP address**, and select **RestAPI** as the connection type within the Home Assistant integration. If authentication is required, Home Assistant will prompt you to enter Smart Gateway's **credentials**. All devices configured on Smart Gateway will be automatically added to your Home Assistant.
```

File: source/_integrations/iskra.markdown [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+
+### Modbus TCP Connection
+
+If your device supports a direct internet connection, such as PQ meters (iMC/MC series/ iMT/MT series usually), you can use Modbus TCP:
+
+- **Device**: Find your device using the [MiQen](https://www.iskra.si/sl/Programska-oprema/MiQen/) software and configure it to use static IP.
+- **Home Assistant**: Add the iskra integration, enter the devices's **IP address** and select **Modbus TCP** as the connection type within the Home Assistant integration. Home Assistant will prompt you to enter Modbus TCP port and Modbus address of your device.
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]:
```suggestion
- **Home Assistant**: Add the Iskra integration, enter the devices's **IP address**, and select **Modbus TCP** as the connection type within the Home Assistant integration. Home Assistant will prompt you to enter the Modbus TCP port and Modbus address of your device.
```

File: source/_integrations/iskra.markdown [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+### Modbus TCP connection
+
+If your device supports a direct internet connection, such as PQ meters (iMC/MC series/ iMT/MT series usually), you can use Modbus TCP:
+
+- **Device**: Find your device using the [MiQen](https://www.iskra.si/sl/Programska-oprema/MiQen/) software and configure it to use static IP.
+- **Home Assistant**: Add the Iskra integration, enter the devices's **IP address**, and select **Modbus TCP** as the connection type within the Home Assistant integration. Home Assistant will prompt you to enter the Modbus TCP port and Modbus address of your device.
+
+{% include integrations/config_flow.md %}
+
+## Monitored data
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown]: are these the sensors? Then, the title should be renamed to Sensors.
In any case, please add a section called Sensors that lists and describes all the sensors that this integration provides.

File: source/_integrations/iskra.markdown [hunk 10]
```diff
@@ -0,0 +1,89 @@
+---
+title: Iskra
+description: Instructions on how to connect your Iskra energy meters to Home Assistant.
+ha_release: 2024.6
```
Comment (joostlek) [inline source/_integrations/iskra.markdown]:
```suggestion
ha_release: "2024.10"
```

File: source/_integrations/iskra.markdown [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+
+These devices typically support Ethernet connections and use Modbus TCP for data polling:
+
+- iMT/MT series ([MTXXX / iMTXXX](https://www.iskra.eu/en/NEW_SERIES_Universal_measuring_devices_/))
+- iMC/MC series ([MCXXX / iMCXXX](https://www.iskra.eu/en/NEW_SERIES_Universal_measuring_devices_/))
+
+## Configuration options
+
+There are two ways to configure your devices with Home Assistant:
+
```
Comment (c0ffeeca7) [inline source/_integrations/iskra.markdown:42]:
```suggestion
- [Using a Smart Gateway with REST API](#smart-gateway-with-rest-api)
- [Using a Modbus TCP connection](#modbus-tcp-connection)

```
I would list them in the introduction, before going into the content.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @iskrakranj :+1:

**Pull request merged**